packages/component-base/src/overlay-class-mixin.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2023 Vaadin Ltd.
```

Comment:
There are 2 new files, one has `2021 - 2023`, and the other has `2023`

Reply:
Thanks, changed to use `2023` for both.